src/index.ts

```diff
@@ -1,5 +1,6 @@
 import * as db from './db';
 import * as App from './app';
 import * as types from './types';
+import * as middleware from './middlewares';
```

Comment:
Will this yield `middleware.middlewares`?

Reply:
changed